### PR TITLE
MM-32230: Clarify mmctl vs cli usage in HA

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/docs/issues/4120

Updated:
- Self-Managed Admin Guide > Scaling Mattermost > High Availability Cluster (E20) > Configuration and Compatibility
   - Added a new subsection called "CLI and High Availability"

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@ Command Line Tools
 
 From the directory where the Mattermost server is installed, a ``mattermost`` command is available for configuring the system. For an overview of the Mattermost command line interface (CLI), `read this article <https://medium.com/@santosjs/plugging-in-to-the-mattermost-cli-8cdcef2bd1f6>`__ from Santos.
 
+.. note::
+
+  When running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a `High Availability environment <https://docs.mattermost.com/deployment/cluster.html>`__, performing tasks such as creating and deleting users or changing configuration settings will require a server restart. This is because the CLI is run in a single node which bypasses the mechanisms that a High Availability environment uses to perform actions across all nodes in the cluster.
```
Comment (justinegeffen) [inline source/administration/command-line-tools.rst:8]:
```suggestion
  The CLI is run in a single node which bypasses the mechanisms that a `High Availability environment <https://docs.mattermost.com/deployment/cluster.html>`__ uses to perform actions across all nodes in the cluster. As a result, when running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a High Availability environment, tasks such as creating and deleting users or changing configuration settings require a server restart.
```

File: source/deployment/cluster.rst [hunk 2]
```diff
@@ -339,6 +339,13 @@ If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` `
 
 Note a slight behavior change in 5.15: When you reinstall a plugin in 5.14, the previous **Enabled** or **Disabled** state is retained. As of 5.15, a reinstalled plugin's initial state is **Disabled**.
 
+CLI and High Availability
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a High Availability environment, performing tasks such as creating and deleting users or changing configuration settings will require a server restart. This is because the CLI is run in a single node which bypasses the mechanisms that a High Availability environment uses to perform actions across all nodes in the cluster.
```
Comment (justinegeffen) [inline source/deployment/cluster.rst:345]: If my above suggestion is workable, then it would replace this one as well.

**Review by justinegeffen — APPROVED** [review]
I've suggested some rewording which is non-blocking. Otherwise this is great. :)

File: source/deployment/cluster.rst (hunk 2 above)
Comment (cwarnermm) [reply source/deployment/cluster.rst:345]: Thanks! Your proposal is a better, clearer message.

**Review by justinegeffen — APPROVED** [review]

**Review by wiersgallak — APPROVED** [review]
Thanks!

File: source/administration/mmctl-cli-tool.rst [hunk 3]
```diff
@@ -3,12 +3,13 @@ mmctl Command Line Tool (Beta)
...
+Being installed locally enables System Admins for both self-managed and Cloud Mattermost instances to run CLI commands even in instances where there is no access to the server (e.g., via SSH). This tool is currently in beta and can be used alongside the Mattermost CLI tool. In the future, the Mattermost CLI tool will be deprecated.
 
 This feature was developed to a large extent by community contributions and we'd like to extend our gratitude to the contributors who have worked on this project. We are currently accepting pull requests for Help Wanted issues in the `mattermost-server <https://github.com/mattermost/mattermost-server/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22+label%3AArea%2Fmmctl>`__ repo. You can learn more about the unit test coverage campaign for mmctl in the `Unit testing mmctl commands <https://mattermost.com/blog/unit-testing-mmctl-commands/>`__ blog post.
 
 **Notes**
 
+-  System Admins have two ways to run `mmctl` commands: by downloading `mmctl` from the repository, or by building it directly. See the `mmctl readme <https://github.com/mattermost/mmctl#install>`__ for details.
```
Comment (mgdelacroix) [inline source/administration/mmctl-cli-tool.rst:12]: Maybe it's worth mentioning as well that `mmctl` comes bundled with the Mattermost distribution that is located in the `bin` folder of the installation, alonside of the `mattermost CLI`

File: source/deployment/cluster.rst [hunk 4]
```diff
@@ -339,6 +339,13 @@ If ``"DriverName": "local"`` is used then the directory at ``"FileSettings":`` `
 
 Note a slight behavior change in 5.15: When you reinstall a plugin in 5.14, the previous **Enabled** or **Disabled** state is retained. As of 5.15, a reinstalled plugin's initial state is **Disabled**.
 
+CLI and High Availability
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The CLI is run in a single node which bypasses the mechanisms that a `High Availability environment <https://docs.mattermost.com/deployment/cluster.html>`__ uses to perform actions across all nodes in the cluster. As a result, when running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a High Availability environment, tasks such as creating and deleting users or changing configuration settings require a server restart.
```
Comment (mgdelacroix) [inline source/deployment/cluster.rst:345]: Creating users should not require a server restart, so I wouldn't use it as an example. I'd suggest to use use updating instead:

```suggestion
The CLI is run in a single node which bypasses the mechanisms that a `High Availability environment <https://docs.mattermost.com/deployment/cluster.html>`__ uses to perform actions across all nodes in the cluster. As a result, when running `CLI commands <https://docs.mattermost.com/administration/command-line-tools.html>`__ in a High Availability environment, tasks such as updating and deleting users or changing configuration settings require a server restart.
```

**Review by mgdelacroix — COMMENTED** [review]
Looks great! A couple of minor changes suggested

File: source/administration/mmctl-cli-tool.rst (hunk 3 above)
Comment (cwarnermm) [reply source/administration/mmctl-cli-tool.rst:12]: This is a great suggestion. Thank you!

**Pull request merged**